easybuild/easyconfigs/o/ORCA/ORCA-5.0.4-gompi-2022a.eb

```
@@ -0,0 +1,23 @@
+name = 'ORCA'
+version = '5.0.4'
+
+homepage = 'https://orcaforum.kofo.mpg.de'
+description = """
+ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum
+chemistry with specific emphasis on spectroscopic properties of open-shell
+molecules. It features a wide variety of standard quantum chemical methods
+ranging from semiempirical methods to DFT to single- and multireference
+correlated ab initio methods. It can also treat environmental and relativistic
+effects."""
+
+toolchain = {'name': 'gompi', 'version': '2022a'}
+
+# Shared build of ORCA: download from https://orcaforum.kofo.mpg.de
```

We could make use of `download_instructions` here, maybe something like:

```suggestion
download_instructions = "Shared build of ORCA: download from https://orcaforum.kofo.mpg.de"
```

---

Done, thanks!